src/main/java/duke/logic/Storage.java

```diff
@@ -0,0 +1,119 @@
+package duke.logic;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+
+/**
+ * Represents a Duke storage that deals with
+ * loading tasks from the file and saving tasks in the file.
```

Comment:
I believe these 2 lines can be combined in a more optimal way instead of 2 abrupt points. Resolve this if I am wrong as I am unable to see the character space limit from github (= 

E.g.
 * Represents a Duke storage that deals with loading tasks from the file and | <<character limit>>
 * saving tasks in the file.